packages/application-generic/src/factories/chat/handlers/guilded.handler.ts

```diff
@@ -0,0 +1,16 @@
+import { ChannelTypeEnum } from '@novu/shared';
+import { GuildedChatProvider } from '@novu/guilded';
+import { ICredentials } from '@novu/shared';
+import { BaseChatHandler } from './base.handler';
+
+export class GuildedHandler extends BaseChatHandler {
+  constructor() {
+    super('guilded', ChannelTypeEnum.CHAT);
+  }
+
+  buildProvider(_credentials: ICredentials) {
+    this.provider = new GuildedChatProvider({});
```

Comment:
I think you do not need to provide the empty object because it says `src/factories/chat/handlers/guilded.handler.ts(12,45): error TS2554: Expected 0 arguments, but got 1.` in ci errors.

Reply:
Okay let me try